src/lib/components/Ideogram.react.js

```diff
@@ -641,4 +609,43 @@ Ideogram.propTypes = {
      * e.g. for mitochondrial (MT) and chloroplast (CP) DNA.
      */
     showNonNuclearChromosomes: PropTypes.bool,
+
+    /**
+     * Callback function to invoke after chromosome has rotated. (React)
+     * onDidRotate: PropTypes.func,
+     */
+
+    /**
+     * Dash event callback for mousing over data.
+     * onMouseOver: PropTypes.func,
+     */
+
+    /**
+     * Callback function to invoke when brush moves.
+     * onBrushMove: PropTypes.func,
+     */
+
+    /**
+     * Callback function to invoke when annotations are drawn. (React)
+     *  onDrawAnnots: PropTypes.func,
```

Comment:
```suggestion
     * onDrawAnnots: PropTypes.func,
```

Reply:
I will skip this one as this prop will probably not be implemented as it is to feed a react function to the component, so it is likely to be erased in the future :)